### DISCUSSION TITLE
Improve contributor onboarding document (README/CONTRIBUTING/Dev notes)

### DISCUSSION THREAD

**Discussion by butchtm** [thinkingmachines/geowrangler#95] [Enhancements Requests]
Centralize the placement of the instructions and clarify the steps for onboarding new contributors:
    - Forking and cloning the repo (including syncing updates after the fork)
    - Setting up a local jupyter notebook dev 
    - (optionally) Setting up jekyll for local doc site and viewing the updated docs locally
    - Making changes to the jupyter notebooks (and updating the code modules)
    - Checking that the modifications pass the project requirements  (formatting, linting, unit tests, notebook tests)
    - Pushing changes back to the repo and making a PR

**Comment by joshuacortez** [comment]
Addressed by #243
